example/android/settings.gradle

```diff
@@ -1,5 +1,9 @@
 rootProject.name = 'example'
 include ':react-native-adjust'
 project(':react-native-adjust').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-adjust/android')
+include ':react-native-adjust'
+project(':react-native-adjust').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-adjust/android')
+include ':react-native-adjust'
```

Comment:
duplicated instruction

Reply:
Yeah, build scripts which are building and running our example app are making sure this happens each time we run them, but having in mind that it's an example app that we mostly use just to demonstrate how SDK API can be used, we're living with it. Will be addressed in future, but for now, it's like this. But fair point. 👍